kornia/utils/grid.py

```diff
@@ -28,12 +28,13 @@ def create_meshgrid(
     # generate coordinates
     xs: Optional[torch.Tensor] = None
     ys: Optional[torch.Tensor] = None
+
+    xs = torch.linspace(0, width - 1, width, device=device, dtype=torch.float)
+    ys = torch.linspace(0, height - 1, height, device=device, dtype=torch.float)
+    # Fix TracerWarning
     if normalized_coordinates:
-        xs = torch.linspace(-1, 1, width, device=device, dtype=torch.float)
-        ys = torch.linspace(-1, 1, height, device=device, dtype=torch.float)
-    else:
-        xs = torch.linspace(0, width - 1, width, device=device, dtype=torch.float)
-        ys = torch.linspace(0, height - 1, height, device=device, dtype=torch.float)
+        xs = (xs / (width - 1) - 0.5) * 2
+        ys = (ys / (height - 1) - 0.5) * 2
```

Comment:
a better option would be to reuse[`K.normalize_pixel_coordinates`](https://kornia.readthedocs.io/en/latest/geometry.conversions.html#kornia.geometry.conversions.normalize_pixel_coordinates)

Reply:
This will not suppress the ```TracerWarning``` since it made new tensors internally. I would keep using the current implementations.

Also, I think we will need to try getting rid of new tensors in our functions.

Reply:
ok, then we need to fix `K.geometry.normalize_pixel_coordinates`, right ?

Reply:
Theoretically, yes. The best way of addressing the JIT TracerWarnings is to initialize **NO NEW TENSORS** inside any functions, or it may lose track of the tensor trace. While I think it is not really achievable under our current library concept, since we wish to handle the python scalars for our params.

Reply:
right, a bit difficult for us. However, from graph perspective I don't it will be a problem since usually new tensors are constants and for backprop I don't think it's a big deal.